CharlieBackend.Business/Services/HomeworkStudentService.cs

```diff
@@ -137,14 +165,14 @@ public async Task<Result<HomeworkStudentDto>> UpdateHomeworkFromStudentAsync(Hom
                     PublishingDate = DateTime.UtcNow,
                 };
 
-                var AttachmentOfHomeworkStudentsHistory = foundStudentHomework.AttachmentOfHomeworkStudents.Select(attachmentOfHomework => new AttachmentOfHomeworkStudentHistory()
+                var AttachmentOfHomeworkStudentsHistory = foundStudentHomework.AttachmentOfHomeworkStudents?.Select(attachmentOfHomework => new AttachmentOfHomeworkStudentHistory()
```

Comment:
Please update the variable according to code style. (should start with lowercase)